src/client/dfuse/ops/open.c

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2016-2024 Intel Corporation.
+ * (C) Copyright 2016-2025 Intel Corporation.
```

Comment:
This should also be 2024 and add a google one. If you pull in latest google/2.6, it will do this for you and stop changing intel one to 2025